### DISCUSSION TITLE
使用路由 /bilibili/followings/video/，提示：对应 uid 的 Bilibili 用户的 Cookie 已过期，但我换浏览器重新登录然后再抓 cookie 还是不行

### DISCUSSION THREAD

**Discussion by TPAXcc** [DIYgod/RSSHub#10292] [Q&A] [answered]
Route requested: /followings/video/1465080156

Error message: 对应 uid 的 Bilibili 用户的 Cookie 已过期

Helpful Information to provide when opening issue:
Path: /followings/video/1465080156
Node version: v16.16.0
Git Hash: f0c5610
`sudo docker run -d --name RSSHUB-Test -p 1200:1200 -e BILIBILI_COOKIE_1465080156="_uuid":"FF1046167-C31F-99E10-A7DA-59D5CAF31BE823434infoc","b_lsid":"2D9E41053_18222C0A56D","b_timer":"{\"ffp\":{\"333.976.fp.risk_A5D74C53\":\"1822023BC72\",\"333.788.fp.risk_A5D74C53\":\"1822023D5AB\",\"333.1007.fp.risk_A5D74C53\":\"182221BB7F5\",\"333.337.fp.risk_A5D74C53\":\"18204C8418F\",\"333.999.fp.risk_A5D74C53\":\"18222CA5CCF\",\"333.937.fp.risk_A5D74C53\":\"181EE2EA60C\",\"333.47.fp.risk_A5D74C53\":\"18204C82C0F\"}}","b_ut":"5","bili_jct":"6b81da49ecf6f0608efea739c5a7f66d","blackside_state":"0","bp_video_offset_1465080156":"677199115892817900","bsource":"search_bing","buvid_fp":"b83301be8f6a71b8bd705bf78bb59f1b","buvid_fp_plain":"undefined","buvid3":"A5D74C53-52D7-4983-B95F-5A7DDC3A40DB25872infoc","buvid4":"D8814028-A235-2DEB-DCF7-3D5D3B16653C30574-022060512-Mk4wjKcJQ46BQpZ1av6yZZI9aAu89A8Sxy+sHQE4EGrqotJFIAoAlQ==","CURRENT_BLACKGAP":"0","CURRENT_FNVAL":"4048","CURRENT_QUALITY":"120","DedeUserID":"1465080156","DedeUserID__ckMd5":"e9b0279be7c05faf","fingerprint":"b83301be8f6a71b8bd705bf78bb59f1b","i-wanna-go-back":"-1","innersign":"0","l":"v","nostalgia_conf":"-1","PVID":"1","rpdid":"0zbfAHOYUy|5CtRucDC|2It|3w1NXHxN","SESSDATA":"799aaae0,1673603387,75841*71","sid":"7zb7sg7u" diygod/rsshub`

**Comment by TPAXcc** [comment]
@DIYgod 救救孩子吧

**Comment by DIYgod** [comment, marked as answer]
你这格式看起来不对，cookie 是这里的
<img width="648" alt="image" src="https://user-images.githubusercontent.com/8266075/185253567-3f32e92f-6731-414a-b621-cc61626a61ab.png">

**Reply by TPAXcc** [reply]
![图片](https://user-images.githubusercontent.com/91980905/185279113-d9fe1de4-334e-435f-9de1-8f496aff33bf.png)
不是这个吗，我只看到这一个请求啊

**Reply by TPAXcc** [reply]
我用新获取的 cookie 已经能运行了，之前是有问题一直报错退出，但是我访问
http://home.pc.u22.04.tpax.xyz:1200/bilibili/followings/dynamic/1465080156
的时候又提示这个:

```
Route requested: /followings/dynamic/1465080156

Error message: Invalid character in header content ["cookie"]: target website might be blocking our access, you can [host your own RSSHub instance](https://docs.rsshub.app/install/) for a better usability.

Helpful Information to provide when opening issue:
Path: /followings/dynamic/1465080156
Node version: v16.17.0
Git Hash: 1a9fc89
```

**Reply by TPAXcc** [reply]
`sudo docker run -d --name RSShub-Test -p 1200:1200 -e BILIBILI_COOKIE_1465080156="----" diygod/rsshub`

**Reply by TPAXcc** [reply]
正常应该不会在我自建的实例提示这个这个 Error message 吧
